cpp_linter/loggers.py

```diff
@@ -53,3 +55,33 @@ def log_response_msg(response: Response):
             response.request.url,
             response.text,
         )
+
+
+def worker_log_file_init(temp_dir: str, log_lvl: int):
```

Comment:
for some reason this function is never executed in unit tests. See [coverage results](https://app.codecov.io/gh/cpp-linter/cpp-linter/pull/92/blob/cpp_linter/loggers.py)

Reply:
I think the separate processes are messing with the code coverage analysis. The `_run_on_single_file()` function is [not showing coverage either](https://app.codecov.io/gh/cpp-linter/cpp-linter/pull/92/blob/cpp_linter/clang_tools/__init__.py).

Reply:
Here is the [coverage docs about measuring subprocesses](https://coverage.readthedocs.io/en/7.4.4/subprocess.html). Not sure if that is helpful. There's also [this SO suggestion](https://stackoverflow.com/a/71023304), but we use the pyproject.toml to configure the coverage tool:
https://github.com/cpp-linter/cpp-linter/blob/005c30b0c7a989bde002bc3138ec5dcf131d6c22/pyproject.toml#L67-L76

Reply:
Now I see that you pushed some changes to help measure the code coverage. Well done! 💯